test/testcafe/framework/page-objects/components/BaseFormObject.js

```diff
@@ -205,6 +205,10 @@ export default class BaseFormObject {
   }
 
   getErrorBoxCount() {
+    if (userVariables.v3) {
+      return within(this.el).getAllByRole('alert').nth(0).count;
```

Comment:
My only feedback for this is that, when you use `getAllByRole(...).nth(n)` it will always only give back the first item that was found (if any), so this function (for v3) will only ever give back 0 or 1. 

Could the test be updated to search for a specific error message instead (both for v2 and v3)?

Reply:
In the 2 specs that this method is used, one of them verifies that the error count is 0. 
The other one verifies that the error count is 1 and then also verifies the error message like this - 
```
  await t.expect(challengePasswordPage.form.getErrorBoxCount()).eql(1);
  await t.expect(challengePasswordPage.form.getErrorBoxText())
    .eql('Reset password is not allowed at this time. Please contact support for assistance.');
```
The error box count is redundant in the above case. But no harm in keeping it I guess? Wdty?

Reply:
Upon further examination, instead of using `getAllByRole(...).nth(n)`, I can use `getAllByRole('alert').count`.
This should give us the desired error count.

Reply:
Ahh, ok that makes much more sense in this case. Ty!